detector/cve/cve20258088/cve20258088.go

```diff
@@ -0,0 +1,170 @@
+var (
+	winrarNameRegex = regexp.MustCompile(`(?i)\b(winrar|unrar)\b|\brar\b`)
+)
+
+const (
+	// Name of the detector.
+	Name = "cve/cve-2025-8088"
+)
+
+// Detector is a SCALIBR Detector for CVE-2025-8088.
+type Detector struct{}
+
+// New returns a detector.
+func New() detector.Detector {
+	return &Detector{}
+}
+
+// Name of the detector.
+func (Detector) Name() string { return Name }
+
+// Version of the detector.
+func (Detector) Version() int { return 0 }
+
+// Requirements of the detector.
+func (Detector) Requirements() *plugin.Capabilities {
+	return &plugin.Capabilities{OS: plugin.OSWindows, DirectFS: true, RunningSystem: true}
+}
+
+// RequiredExtractors of the detector.
+func (Detector) RequiredExtractors() []string {
+	return []string{"windows/ospackages", "windows/peversion"}
+}
+
+// DetectedFinding returns generic vulnerability information about what is detected.
+func (d Detector) DetectedFinding() inventory.Finding {
+	return d.findingForPackage(nil)
+}
+
+func (d Detector) findingForPackage(dbSpecific map[string]any) inventory.Finding {
+	pkg := &extractor.Package{
+		Name:     "winrar",
+		PURLType: "generic",
+	}
+	vuln := d.makePackageVulnWithDb(pkg, "7.13", "WinRAR path traversal vulnerability (<7.13)",
+		"WinRAR versions before 7.13 are vulnerable to a path traversal attack that allows attackers to extract files to arbitrary locations outside the intended extraction directory.",
+		dbSpecific)
+	return inventory.Finding{PackageVulns: []*inventory.PackageVuln{vuln}}
+}
+
+// Scan checks for the presence of the WinRAR RCE CVE-2025-8088 vulnerability.
+func (d Detector) Scan(ctx context.Context, scanRoot *scalibrfs.ScanRoot, px *packageindex.PackageIndex) (inventory.Finding, error) {
+	var findings []*inventory.PackageVuln
+
+	// === Phase 1: Installed packages via package index ===
+	if px == nil {
+		log.Infof("cve20258088: PackageIndex is nil, no packages to scan")
+		return inventory.Finding{}, nil
+	}
+
+	allPackages := px.GetAll()
+	log.Infof("cve20258088: Scanning %d packages from PackageIndex", len(allPackages))
+
+	for _, pkg := range allPackages {
+		log.Debugf("cve20258088: Checking package %q version %q", pkg.Name, pkg.Version)
+
+		// Use word boundary regex to avoid false positives like "Libraries", "Hardware", etc.
+		if !winrarNameRegex.MatchString(pkg.Name) {
+			continue
+		}
+
+		log.Debugf("cve20258088: Package %q matched WinRAR regex", pkg.Name)
+
+		normalizedVersion := normalizeVersion(pkg.Version)
+		if normalizedVersion == "" {
+			continue
+		}
+
+		if sv, err := semantic.Parse(normalizedVersion, "Maven"); err == nil {
+			if cmp, err := sv.CompareStr("7.13"); err == nil && cmp < 0 {
+				log.Infof("cve20258088: Vulnerable WinRAR package found: Package %q, Version: %q",
+					pkg.Name, normalizedVersion)
+
+				details := fmt.Sprintf("WinRAR %s detected via package index", normalizedVersion)
+				vuln := d.makePackageVuln(pkg, normalizedVersion, "WinRAR vulnerable version detected", details)
```

Comment:
Details needs to be the same for all instances of this vulnerability - I'd advise to just use findingForPackage() with a dbspecific value set for setting up the vuln here too.